api.rb

```diff
@@ -127,6 +127,10 @@ def on_close(room, q)
       r.halt 404 unless (game = Game[id])
       render(game_data: game.to_h(include_actions: true))
     end
+
+    r.on 'all_tiles' do
+      render(app_route: 'all_tiles')
```

Comment:
you can just put this in the main  on line 94

Reply:
3c5cb1b